couchbase/go.mod

```diff
@@ -0,0 +1,12 @@
+module github.com/gofiber/storage/couchbase
+
+go 1.20
```

Comment:
This should be `1.18`, like the official one https://github.com/couchbase/gocb/blob/master/go.mod